spec/bolt/node/ssh_spec.rb

```diff
@@ -21,6 +21,7 @@ def mk_config(conf)
   let(:command) { "pwd" }
   let(:config) { mk_config(user: user, password: password) }
   let(:insecure) { mk_config(insecure: true, user: user, password: password) }
+  let(:disable_ssh_agent) { { config: Bolt::Config.new(transports: { ssh: { 'disable-ssh-agent' => true } }) } }
```

Comment:
This is unused, did you intend to add a test using it?
Also you could make it shorter using the the `mk_config` function.

Reply:
I did, thanks.

Reply:
Working on a valid test ...